### DISCUSSION TITLE
Repeatedly seeking backwards in the editor when a spinner is visible causes bonus points to accumulate

### DISCUSSION THREAD

**Discussion by nagi-desuuu** [ppy/osu#20207] [Q&A] [answered]
As the title would suggest, going back from the area before a spinner starts wouild result in points getting stacked from previous spinner's "playthroughs"
I don't really have the proper choice of words for this, so here's a video reproducing this cosmetic issue in version 2022.902.1 of osu!lazer.
https://streamable.com/ito5fa

**Comment by Gabixel** [comment, marked as answer]
Related issue: https://github.com/ppy/osu/issues/10455
